spec/12-the-scala-standard-library.md

```diff
@@ -642,6 +642,12 @@ object Predef {
   def classOf[T]: Class[T] = null
    // this is a dummy, classOf is handled by compiler.
 
+  // valueOf -----------------------------------------------------------
+
+  /** Retrieve the single value of a type with a unique inhabitant. */
+  @inline def valueOf[T](implicit vt: ValueOf[T]): T {} = vt.value
```

Comment:
I think perhaps it would be appropriate to add a comment to `classOf`, explaining (or trying to explain) why it don't have the obvious implementation.